skrf/vi/__init__.py

```diff
@@ -1,164 +1,69 @@
-
-If a command is read or write only, then you specify only the query or set items.  If it is read/write, then you
-specify both.  query_values is just a query command that returns a data set, and will then use the pyvisa
-query_values convenience function that will automatically parse the returned data and place it in a list or numpy
-array of floats.
-
-The set/query items are merely the string of arguments that must be provided to the command.  For example, the
-'create_meas' example above looked like this:
-
-::
-
-    :CALC1:PAR:DEF:EXT 'my_meas','S11'
-
-If we placed this command in a python function and called it, it might look something like this:
-
-::
-
-    class SCPI:
-        def __init__(self, pyvisa_resource):
-            self.resource = pyvisa_resource
-
-        def set_create_meas(self, cnum=1, mname="", param="")
-            scpi_command = ":CALC{:}:PAR:DEF:EXT '{:}','{:}'".format(cnum, mname, param)
-            self.resource.write(scpi_command)
-
-    vna = SCPI(pyvisa_resource)
-    vna.set_create_meas(1, "my_meas", "S11")
-
-This illustrates what is going on with the yaml file.  If we isolate only the create_meas command in the command tree
-the yaml mapping looks like this:
-
-::
-
-    CALC<cnum=1>:
-      PAR:
-        DEF:
-          EXT: {name: create_meas, set: "'<mname>','<param>'"}
-
-the python function ``set_create_meas`` is simply the write method of the create_meas SCPI command.  The 'set' item
-in the yaml mapping is the string: ``"'<mname>','<param>'"`` which is the the required arguments for the SCPI command.
-The function has three keyword arguments, cnum, mname and param which are enclosed in <> brackets.  If an argument
-has a default parameter (int, float or string) then it can be specified with an '=' sign.
-
-Using the SCPI functions to write a Driver
----------------------------------------------------
-
-The parser outputs a python file with a SCPI class.  The current model for using this class is to initialize a VNA
-class, which in turn initialized a pyvisa resource.  The VNA object then initializes a SCPI object and assigns it to
-self.scpi.  See the abbreviated code from the keysight_pna.py PNA class __init__ method:
-
-::
-
-    def __init__(self, address=DEFAULT_VISA_ADDRESS, **kwargs):
-        super(PNA, self).__init__(address, **kwargs)
-        self.resource.timeout = kwargs.get("timeout", 2000)
-        self.scpi = keysight_pna_scpi.SCPI(self.resource)
-
-Subsequently, the low-level instrument commands are contained with the .scpi namespace and can be called in the
-following manner inside vna object methods:
-
-::
-
-    f_start = self.scpi.query_f_start(channel)
-    self.scpi.set_f_start(channel, f_start)
-
-
-
+=================
+
+Drivers are classes that from the respective instrument type's abstract base
+class and define methods specfic to each instrument. This syntax can be complex
```

Comment:
I don't understand this sentence. Is this suggestion means the same?

```suggestion
Drivers are classes inherited from the respective instrument type's abstract base
class and define methods specific to each instrument. This syntax can be complex
```

Reply:
The original had typos :)